pro-duck-store/src/App.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './App.css';
+
+
+import MiniDrawer from "./components/Drawer/drawer";
+import {Store} from "./pages/store";
+import {BrowserRouter, Route, Switch} from "react-router-dom";
+import {Cart} from "./pages/cart";
+import {User} from "./pages/user";
+
+
+function App() {
+  return (
+      <BrowserRouter>
+      <MiniDrawer/>
+      <main style={{display: 'flex', paddingLeft: '80px'}}>
+          <Switch>
```

Comment:
What if the user is directly accessing a path that you don't have in your switch? For example http://localhost:4200/asdfg. For such cases you need to have a wildcard route that catches all the paths that do not fit with any other declared route and return a 404 page.